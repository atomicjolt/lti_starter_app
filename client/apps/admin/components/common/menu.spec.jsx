import React from 'react';
import TestRenderer from 'react-test-renderer';

import Menu from './menu';

describe('Menu', () => {
  it('renders the Menu component', () => {
    const result = TestRenderer.create(
        <Menu>
          {(onClick, activeClass, isOpen, menuRef) => (
            <div className={activeClass} ref={menuRef}>
              <button
                onClick={onClick}
                type="button"
              >
                <i className="material-icons" aria-hidden="true">more_vert</i>
              </button>
              <div className={`${isOpen ? 'open' : 'closed'}`}></div>
            </div>
          )}
          </Menu>
    );
    expect(result).toMatchSnapshot();
  });
});
