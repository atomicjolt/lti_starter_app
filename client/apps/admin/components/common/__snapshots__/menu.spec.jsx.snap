// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders the Menu component 1`] = `
<div
  className=""
>
  <button
    onClick={[Function]}
    type="button"
  >
    <i
      aria-hidden="true"
      className="material-icons"
    >
      more_vert
    </i>
  </button>
  <div
    className="closed"
  />
</div>
`;
